Friends/friendRouter.js

```diff
@@ -0,0 +1,88 @@
+const router = require('express').Router();
+const Friend = require('./friendModel');
+
+
+router
+    .route('/')
+    .get((req, res) => {
+        Friend.find()
+            .then(friendsArray => {
+                res.status(200).json({ friendsArray });
+            })
+            .catch(err => {
+                res.status(500).json({ errorMessage: "The friends information could not be retrieved." });
+            })
+    })
+    .post((req, res) => {
```

Comment:
Nicely done with this POST route! Only suggestion would be to use object destructuring on `req.body`, so you don't have to repeat `friendData` so many times:

```
const { firstName, lastName, age } = req.body

if (!firstName || !lastName || !age) {
  // ...
} else if (isNaN(age) || age < 1 || age > 120) {
  // ...
}
```